distributed/scheduler.py

```diff
@@ -6785,6 +6924,7 @@ def workers_to_close(
         minimum: "int | None" = None,
         target: "int | None" = None,
         attribute: str = "address",
+        stimulus_id=None,
```

Comment:
This is only a "get information" method and is not modifying anything. The stimulus IDs are basically only relevant for mutating methods

Reply:
removed